cloudfour/cloudfour.com-patterns
# Use logical properties consistently

## Overview

This PR installs a plugin for linting logical properties and applies those fixes across all existing stylesheets.

Please note that the intent of this PR is to enforce consistent property usage, _not_ to optimize each and every component for different text directions and writing modes.

## Screenshots

<img width="1260" alt="Screen Shot 2021-11-22 at 3 02 16 PM" src="https://user-images.githubusercontent.com/69633/142947859-85bb4408-ae2c-4be9-b9d1-283c19dcc3ec.png">

## Testing

1. View a few patterns with "text flow" set to something other than the default to verify that properties have been updated.
2. Observe basically every pattern in the deploy preview for regressions using the "default" text flow. I would most appreciate 👀 on Chrome, Edge and/or Safari (as Firefox is my primary browser and probably has disproportionate coverage).

---

- Fixes #1576
